docs/getting-started/use-cases/computervision/annotation/cvat/cvat_customizing_workspace_template.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Customizing CVAT Workspace Template
+sidebar_label: Environment variables - CVAT Workspace
+description: List of CVAT Workspace environment variables.
+---
+
+These environment variables were added to our CVAT image.
```

Comment:
I would change these two sentences into: "You can update the default CVAT template adjust the following environment variables to change where CVAT data is stored."

Reply:
Done